gnat2goto/driver/driver.adb

```diff
@@ -143,7 +143,7 @@ package body Driver is
    end GNAT_To_Goto;
 
    procedure Initialize_CProver_Internal_Variables (Start_Body : Irep) is
-      Int_32_T : constant Irep := Make_Signedbv_Type (Ireps.Empty, 32);
+      Int_32_T : constant Irep := Make_Signedint_Type (32);
```

Comment:
`Make_SignedInt_Type` .. Capital I

Reply:
Nope. Consistent with: Make_Signedbv not Make_SignedBv

Reply:
I think the function definition is spelled with capital I.

Reply:
Fixed. Thanks.